mynav.cc

```diff
@@ -20,39 +20,20 @@
 
  */
 
-#include "defs.h"
 #include <QtCore/QDebug>
 #include <QtCore/QStringList>
+#include <QtCore/QTextStream>
```

Comment:
My hacked up version of include-what-you-use, which is by no means perfect, thinks some include changes are in needed.  I have custom mapping files for Qt.  We usually use a different order: c library includes, Qt includes, defs.h, other gpsbabel includes than iwyu spits out.  We certainly aren't entirely consistent on order.
```
mynav.h should add these lines:
#include <QtCore/QString>  // for QString
#include <QtCore/QVector>  // for QVector

mynav.h should remove these lines:

The full include-list for mynav.h:
#include <QtCore/QString>  // for QString
#include <QtCore/QVector>  // for QVector
#include "defs.h"          // for ff_cap, ff_cap_none, ff_cap_read, CET_CHARSET_ASCII, ff_type, ff_type_file, route_head
#include "format.h"        // for Format
---

mynav.cc should add these lines:
#include <QtCore/qglobal.h>    // for qDebug
#include <QtCore/QChar>        // for operator==, QChar
#include <QtCore/QCharRef>     // for QCharRef
#include <QtCore/QIODevice>    // for QIODevice, QIODevice::ReadOnly

mynav.cc should remove these lines:

The full include-list for mynav.cc:
#include "mynav.h"
#include <QtCore/qglobal.h>    // for qDebug
#include <src/core/file.h>     // for File
#include <QtCore/QChar>        // for operator==, QChar
#include <QtCore/QCharRef>     // for QCharRef
#include <QtCore/QDebug>       // for QDebug
#include <QtCore/QIODevice>    // for QIODevice, QIODevice::ReadOnly
#include <QtCore/QStringList>  // for QStringList
#include <QtCore/QTextStream>  // for QTextStream
```